packages/create-next-app/templates/typescript/next-env.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
 /// <reference types="next/image-types/global" />
```

Comment:
Should this one be moved too?

Reply:
That one is under a conditional so I decided not to include it because of that.